CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ## v0.24.0
-* Update zq to [v0.29.0](https://github.com/brimdata/zq/releases/tag/v0.29.0)
+* Update Zed to [v0.29.0](https://github.com/brimdata/zed/releases/tag/v0.29.0)
```

Comment:
```suggestion
* Update zq to [v0.29.0](https://github.com/brimdata/zed/releases/tag/v0.29.0)
```

I find it a little odd to go through the CHANGELOG and revise history as if Zed was a thing back when it wasn't. So, the link change makes sense to me, but saying "Update Zed" in place of "Update zq" is questionable in my mind. If a user follows the PR links they're going to find a whole lot of Zed-not-being-mentioned which might seem confusing. I respect the spirit of the search & replace exercise though, and I doubt most users are going through super old release notes at this point, so I understand if you wanna still make these changes.

Regardless, I'm only making this comment at the first place I see it in this doc. So if you do roll back the "Zed" mention in the text part, you'd want to do it throughout that file.

Reply:
I think you're right, so I'll change these back to "zq".